package.json

```diff
@@ -34,6 +34,7 @@
     "redux-little-router": "~14.2.1",
     "redux-logger": "~3.0.6",
     "redux-thunk": "~2.2.0",
+    "redux-responsive": "^4.3.5",
```

Comment:
Did you measure performance when resizing the window? :D

Reply:
Haha nope, but I tried resizing and I couldnt notice any performance impacts.
Other than that I don't care about performance when resizing :P

Reply:
does not cause performance impacts as it only pushes to the store when the viewport changes from small to big. Not for every pixel change 👍